demos/backend-workbench/server-py-aiohttp/director_proxy.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+import logging
+import json
+from requests import Session, RequestException
+
+s = Session()
```

Comment:
Does this session works in single-thread servers, i.e. event-loop based server?? or is this supposes to work when this module is imported in different processes?

Reply:
the session object is used to make encrypted requests towards our registry for example. There I still need to look into docker secret stuff to store.
I am not quite sure to understand what you mean there. everytime you want to use it you need to setup the connection first.

Reply:
and added a makefile, too.
But I do not really see the usage of the makefiles at the moment.